src/translations/en.json

```diff
@@ -3227,6 +3227,8 @@
           "description_publish": "Publish a packet",
           "topic": "topic",
           "payload": "Payload (template allowed)",
+          "payload_no_template": "Payload",
```

Comment:
Not sure we want to update the text label with the option switch. We also could change the label to `Payload` (en) and create a new translation key.